src/hooks/Guilds/contracts/useDecodedCall.ts

```diff
@@ -161,24 +161,7 @@ export const decodeCall = async (
       };
   }
   if (!matchedContract) {
-    decodedCall = {
-      callType: SupportedAction.RAW_TRANSACTION,
-      from: call.from,
-      to: call.to,
-      value: call.value,
-      function: null,
-      args: {},
-      optionalProps: {
-        data: call.data,
-      },
-    };
-
-    return {
-      id: `action-${Math.random()}`,
-      decodedCall,
-      contract: null,
-      approval: call.approval || null,
-    };

```

Comment:
Removing this part of the code crashes the page while going to a proposal page with a raw transaction.

Reply:
Yup, definitely didn't mean to commit these. Removing, thanks for catching this :)